### DISCUSSION TITLE
Remove SDL2 from name of release packages

### DISCUSSION THREAD

**Discussion by zenseii** [ihhub/fheroes2#7696] [General]
Hi, @ihhub and @oleg-derevenetz.

Now that we've removed SDL1 support, should we remove the SDL2 part of the names of the releases?
![image](https://github.com/ihhub/fheroes2/assets/12501091/41890568-4944-4ab8-a02f-cb962c1c5e5e)

**Comment by oleg-derevenetz** [comment]
No, because there is SDL3 coming soon.

**Reply by zenseii** [reply]
I see. I can't find any estimation of its release date, for example "No due date" https://github.com/libsdl-org/SDL/milestone/6, only that they officially shifted focus towards it in November last year.

If I understand it correctly then, once it releases soon, we expect to be able to readily add support for it, hence why there is no reason to make this change.

**Reply by oleg-derevenetz** [reply]
It is currently in progress but eventually it will be released, so we shouldn't remove the SDL2 tag just to bring it back later along with SDL3 tag.
